example/android/app/src/google-services.json

```diff
@@ -7,7 +7,7 @@
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "",
+        "mobilesdk_app_id": "1:805978203344:android:351562e0e66401655aaad5",
```

Comment:
Is it safe to commit it to the public repo? Shouldn't this be a secret? :thinking:

Reply:
This is simply an identifier of the app on Firebase, it's not a secret. The whole file can contain some secrets (like `api_key` field), but according to Google, it's safe to keep these files, even with all the data, in the repository. See https://groups.google.com/g/firebase-talk/c/bamCgTDajkw